tests/specs/scss/functions.scss

```diff
@@ -0,0 +1,29 @@
+
+@include test-module('Functions') {
+
+  // Testing set functions
+  @include test('config-set [function]') {
+    @include config-set('test.bar', 'foo');
+    @include config-set('test.color', #fff);
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.